src/hooks/useBsv.ts

```diff
@@ -97,6 +97,18 @@ export type Web3SignMessageRequest = {
   tag?: DerivationTag;
 };
 
+export type Web3EncryptRequest = {
+  message: string;
+  encoding: 'utf8' | 'hex' | 'base64';
+  hexPubKeys: string[];
```

Comment:
I realize the `hex` name is helpful, but that is not in our other APIs. I think the naming should be consistent